shared/tracker/dumb.native.js

```diff
@@ -0,0 +1,225 @@
+  lastTrack: lastTrackMax,
+  proofs: proofsDefault,
+  lastAction: 'followed',
+}, () => true)
+
+const propsWhatevz: TrackerProps = setFollow({
+  ...propsFollowing,
+  reason: 'You have tracked gabrielh',
+  proofs: [
+    proofGithub,
+    {...proofTwitter, meta: metaIgnored},
+  ],
+}, () => true)
+
+const propsChangedProofs: TrackerProps = {
+  ...propsDefault,
+  reason: 'Some of gabrielh\'s proofs have changed since you last tracked them.',
+  userInfo: {
+    ...propsNewUser.userInfo,
+    followsYou: true,
+  },
+  lastTrack: lastTrackMax,
+  trackerState: error,
+  proofs: proofsChanged,
+}
+
+const propsUnfollowed: TrackerProps = {
+  ...propsDefault,
+  reason: 'You have untracked gabrielh.',
+  userInfo: {
+    ...propsNewUser.userInfo,
+    followsYou: true,
+  },
+  lastAction: 'unfollowed',
+}
+
+const propsLessData: TrackerProps = {
+  ...propsBase,
+  nonUser: false,
+  username: '00',
+  reason: 'I\'m a user with not much data.',
+  userInfo: {
+    fullname: 'Hi',
+    bio: '',
+    followersCount: 1,
+    followingCount: 0,
+    followsYou: false,
+    avatar: 'http://placehold.it/140x140/ffffff/000000',
+    location: '',
+  },
+  shouldFollow: true,
+  currentlyFollowing: false,
+  trackerState: normal,
+  proofs: [
+    proofGithub,
+  ],
+}
+
+const propsLoggedOut: TrackerProps = {...propsDefault, loggedIn: false, reason: 'You accessed a public folder with gabrielh.'}
+const propsOneProof: TrackerProps = {...propsDefault, proofs: [proofsDefault[0]]}
+const propsFiveProof: TrackerProps = {
+  ...propsDefault,
+  userInfo: {
+    ...propsDefault.userInfo,
+    bio: 'bio',
+    location: '',
+  },
+  proofs: [0, 1, 2, 3, 4].map(proofGithubMaker),
+}
+
+const dumbMap: DumbComponentMap<Tracker> = {
+  component: Tracker,
+  mocks: {
+    'New user': trackerPropsToRenderProps(propsNewUser),
+    'New user, follows me': trackerPropsToRenderProps(propsNewUserFollowsYou),
+    'Only one proof': trackerPropsToRenderProps(propsOneProof),
+    '5 proofs': trackerPropsToRenderProps(propsFiveProof),
+    'Followed': trackerPropsToRenderProps(propsFollowing),
+    'Changed/Broken proofs user you dont follow': trackerPropsToRenderProps({...propsChangedProofs, lastTrack: null}),
+    'Changed/Broken proofs': trackerPropsToRenderProps(propsChangedProofs),
+    'You track them': trackerPropsToRenderProps({...propsFollowing, userInfo: {...propsNewUser.userInfo, followsYou: false}}),
+    'Unfollowed': trackerPropsToRenderProps(propsUnfollowed),
+    'Barely there': trackerPropsToRenderProps(propsLessData),
+    'Whatevz': trackerPropsToRenderProps(propsWhatevz),
+    'NonuserNoLinkPrivate': trackerPropsToRenderProps({...propsNonUser, inviteLink: null, isPrivate: true}),
+    'NonuserLink': trackerPropsToRenderProps(propsNonUser),
+    'NonuserNoLinkPublic': trackerPropsToRenderProps({...propsNonUser, inviteLink: null}),
+    'Logged out': trackerPropsToRenderProps(propsLoggedOut),
+  },
+}
+
+export default {
+  'Tracker': dumbMap,
+}
+
```

Comment:
If parentProps is the only difference maybe they should be combined and parentProps would be behind a platform check